app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +36,12 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".predator.PreyCatchFragmentTestActivity">
```

Comment:
This activity does not exist in master anymore

Reply:
yes this is resolved with the merging conflicts